system/perses/charts/values.schema.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "object",
+  "additionalProperties": true
+}
```

Comment:
what is this file used for?

Reply:
we do not need this file. This file doesn't do anything useful actually. I have removed it in 3067c4943a43c6ef22ca1cf443b46cb723b29eb7